src/reentrancy.sw

```diff
@@ -7,46 +7,56 @@ use ::context::contract_id;
 use ::auth::caller_is_external;
 use ::option::*;
 
+const SAVED_REGISTERS_OFFSET = 64; // 8 words * 8 bytes
+const CALL_FRAME_OFFSET = 48;      // 6 words * 8 bytes
```

Comment:
Can you put these constants in `constants.sw`?

Reply:
Yes, makes sense.
 I'll take another look at `constants.sw` as well and see if the current constants are all needed. using the `0u8` syntax really makes a lot of hardcoded numbers as constants unnecessary.